src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -268,15 +269,25 @@
      */
     @Argument(fullName="output_model",
             shortName = "outputModel",
-            doc="If specified, the variant recalibrator will output the VQSR model fit to the file specified by -modelFile or to stdout",
+            doc="If specified, the variant recalibrator will output the VQSR model to this file path.",
             optional=true)
-    private boolean outputModel = false;
+    private String outputModel = null;
 
-    @Argument(fullName="model_file",
-            shortName = "modelFile",
-            doc="A GATKReport containing the positive and negative model fits",
+    /**
+     *  The filename for a VQSR model fit to use to recalibrate the input variants. This model should be generated using
+     *  a previous VariantRecalibration run with the --output_model argument.
+     */
+    @Argument(fullName="input_model",
+            shortName = "inputModel",
+            doc="If specified, the variant recalibrator will read the VQSR model from this file path.",
+            optional=true)
+    private String inputModel = null;
+
+    @Argument(fullName="sample_every_Nth_variant",
+            shortName = "sampleEvery",
+            doc="If specified, the variant recalibrator will use only a subset of variants consisting of every Nth variant where N is specified by this argument",
```

Comment:
I thought this approach was bad because then positions get left out of the recal files?  If it is then we should remove this argument.